### PR TITLE
Player ss

### PR DESCRIPTION
**Pull request by sasloan**
@rallen20  Hey boss if you see this before we meet up. Dione helped me figure out our issue. We were making two player boards and calling on the one we did not fire upon. We called on the game method then the player and that fixed it all. I refactored everything. and cleaned up all the files. then i ran all the tests and they all pass and finally I played the game myself and it works great!! please fill free to check it out. but I think we are ready to merge this branch and start work on expanding the board!!!

### CODE REVIEW

File: test/opponent_test.rb [hunk 1]
```diff
@@ -98,16 +100,14 @@ def test_it_knows_a_possible_placement
...
 
   def test_it_can_shoot
-    skip
     board =  "  1 2 3 4 \n" +
              "A . . . . \n" +
              "B . . . . \n" +
              "C . . . . \n" +
              "D . . . . \n"
-    assert board == @game.board.render
+    assert board == @player.board.render
```
Comment (rallen20) [inline test/opponent_test.rb]: @sasloan Hey Sebastian, Once we deleted creating the player instance in the opponent test folder we should only refer to player board through the game. In this case we should use @game.player.board.render as it is in the refute in line 111.

File: test/opponent_test.rb [hunk 2]
```diff
@@ -5,11 +5,13 @@
...
 
+
 class OpponentTest < Minitest::Test
 
   def setup
-    game = "Place Holder"
-    @opponent = Opponent.new(game)
+    @game = Game.new
+    @opponent = Opponent.new(@game)
+    @player = Player.new(@game)
```
Comment (rallen20) [inline test/opponent_test.rb]: This line can be deleted. Player is being called through game. (So @game.player)

Comment (sasloan) [reply test/opponent_test.rb]: got it, makes sense good call .

File: test/opponent_test.rb (hunk 1 above)
Comment (sasloan) [reply test/opponent_test.rb]: @rallen20 I went ahead and refactored those changes. good catch please take a look at the code and if it is what you are looking for then we are ready to merge

**Comment by rallen20** [comment]
Everything looks good. Merging to master.

**Pull request merged**